mk/spksrc.tc.mk

```diff
@@ -8,23 +8,24 @@ include ../../mk/spksrc.directories.mk
 
 
 # Configure the included makefiles
-URLS          = $(TC_DIST_SITE)/$(TC_DIST_NAME)
-NAME          = $(TC_NAME)
-COOKIE_PREFIX = $(TC_NAME)-
+URLS             = $(TC_DIST_SITE)/$(TC_DIST_NAME)
+NAME             = $(TC_NAME)
+COOKIE_PREFIX    = $(TC_NAME)-
 ifneq ($(TC_DIST_FILE),)
-LOCAL_FILE    = $(TC_DIST_FILE)
+LOCAL_FILE       = $(TC_DIST_FILE)
 # download.mk uses PKG_DIST_FILE
-PKG_DIST_FILE = $(TC_DIST_FILE)
+PKG_DIST_FILE    = $(TC_DIST_FILE)
 else
-LOCAL_FILE    = $(TC_DIST_NAME)
+LOCAL_FILE       = $(TC_DIST_NAME)
 endif
-DISTRIB_DIR   = $(TOOLCHAINS_DIR)/$(TC_VERS)
-DIST_FILE     = $(DISTRIB_DIR)/$(LOCAL_FILE)
-DIST_EXT      = $(TC_EXT)
+DISTRIB_DIR      = $(TOOLCHAINS_DIR)/$(TC_VERS)
+DIST_FILE        = $(DISTRIB_DIR)/$(LOCAL_FILE)
+DIST_EXT         = $(TC_EXT)
+TC_LOCAL_VARS_MK = $(WORK_DIR)/tc_vars.mk
 
 #####
 
-RUN = cd $(WORK_DIR)/$(TC_BASE_DIR) && env $(ENV)
+RUN = cd $(WORK_DIR)/$(TC_PREFIX) && env $(ENV)
```

Comment:
Is TC_PREFIX really a folder?

Reply:
Same as above, but now using `TC_TARGET` for directory purpose as `TC_BASE_DIR` was never different anyway.